net-dialup/lrzsz/lrzsz-0.12.20.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="Communication package providing the XMODEM, YMODEM & ZMODEM protocols"
+DESCRIPTION="lrzsz is a Unix package providing the XMODEM, YMODEM AND ZMODEM \
+file transfer protocols."
+HOMEPAGE="https://ohse.de/uwe/software/lrzsz.html"
+COPYRIGHT="1996-1998 Uwe Ohse"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://ohse.de/uwe/releases/lrzsz-0.12.20.tar.gz"
+CHECKSUM_SHA256="c28b36b14bddb014d9e9c97c52459852f97bd405f89113f30bee45ed92728ff1"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	lrzsz = $portVersion
+	cmd:lrz = $portVersion
+	cmd:lsz = $portVersion
+	cmd:lrb = $portVersion
+	cmd:lrx = $portVersion
+	cmd:lsb = $portVersion
+	cmd:lsx = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:ld
+	cmd:make
+	"
+
+PATCH()
+{
+	sed -i -e "s|ln|ln -s|" src/Makefile.in
```

Comment:
Can you please change this to:
```
	sed -i -e "s|ln|ln -r -s|" src/Makefile.in
```
The "-r" (or --relative) will produce nice relative symbolic links (instead of the absolute ones) for the files we install in bin.